lib/sync/worker.js

```diff
@@ -0,0 +1,140 @@
+var syncUtil = require('./util');
+var debug = syncUtil.debug;
+var metrics = require('./sync-metrics');
+var backoff = require('backoff');
+
+function initBackoff(backOffOpts) {
+  var bo;
+  if (backOffOpts.strategy && backOffOpts.strategy.toLowerCase() === 'exp') {
+    bo = new backoff.ExponentialStrategy({initialDelay: backOffOpts.min, maxDelay: backOffOpts.max});
+  } else if (backOffOpts.strategy &&  backOffOpts.strategy.toLowerCase() === 'fib') {
+    bo = new backoff.FibonacciStrategy({initialDelay: backOffOpts.min, maxDelay: backOffOpts.max});
+  } else {
+    //if no valid value is given, just return the default minimum delay. Can be used to turn off backoff.
+    console.log('[warning] no valid strategy found in the backoff options %o. Turning off backoff', backOffOpts);
```

Comment:
Should this include 'sync' somewhere in the log?
e.g. 'No valid sync strategy found in the backoff options %o. Turning off sync worker backoff'